app/src/main/java/com/andrew/tasky/agenda/data/agenda/AgendaRepositoryImpl.kt

```diff
@@ -153,4 +153,11 @@ class AgendaRepositoryImpl(
             // Todo sync events
         }
     }
+
+    override suspend fun deleteAllAgendaTables() {
+        db.getReminderDao().deleteModifiedReminderDb()
+        db.getReminderDao().deleteReminderDb()
+        db.getTaskDao().deleteModifiedTaskDb()
+        db.getTaskDao().deleteTaskDb()
+    }
```

Comment:
You can also just do `db.clearAllTables()`, otherwise you might forget to extend this function when you implement more agenda items in future. Of course this only works if you know you'll always want to clear the whole DB, so if there was some other unrelated DB schema, that could go in its own DB.